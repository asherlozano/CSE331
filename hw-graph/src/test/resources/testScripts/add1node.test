# addNode expected file
# tests if a node can be added to a graph
# a node can be used to create an edge

# Create a graph
CreateGraph g1

# Create a node
AddNode g1 n1

# Print the nodes in the graph
ListNodes g1