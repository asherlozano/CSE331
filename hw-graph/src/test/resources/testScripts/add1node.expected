# addNode expected file
# tests if a node can be added to a graph
# a node can be used to create an edge

# Create a graph
created graph g1

# Create a node
added node n1 to g1

# Print the nodes in the graph
g1 contains: n1